src/daft-connect/src/util.rs

```diff
@@ -0,0 +1,108 @@
+use std::net::ToSocketAddrs;
+
+#[macro_export]
```

Comment:
our other `err!` macros return the error. https://github.com/universalmind303/Daft/blob/ebf964458b42a743ad770ff42fd05d82ff87ce09/src/daft-sql/src/error.rs#L81

Reply:
fixed 3829304186417ccec5a4a178d95ba93819fa92eb